lk2nd/device/dts/msm8909/msm8909-zte-sapphire.dts

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8909 0>;
+	qcom,board-id = <QCOM_BOARD_ID_QRD 0x09>,
+			<QCOM_BOARD_ID(QRD, 1, 0) 0x09>,
+			<0x1000b 110>, <0x1040b 110>; // for sapphire
```

Comment:
```suggestion
	qcom,board-id = <0x1000b 0x6e>, <0x1040b 0x6e>;
```

I believe it's not real SKUE (`0x1000b 0x09` or `0x1000b 0x109`).

Reply:
yeah, that is why I did not add it to the existing skue file